.test/test/pg_database_dialects_test.go

```diff
@@ -1390,7 +1390,7 @@ func TestRdsPgDatabaseDialect_GetBlueGreenStatus(t *testing.T) {
 		QueryerContext: mockQueryer,
 	}
 
-	expectedQuery := "SELECT version, endpoint, port, role, status FROM rds_tools.show_topology('aws_advanced_go_wrapper-1.0.0')"
+	expectedQuery := "SELECT version, endpoint, port, role, status FROM rds_tools.show_topology('aws_advanced_go_wrapper-1.1.0')"
```

Comment:
maybe we should replace this and the other tests in this file to use the constant from driver_version.go